fix(request-validator) properly escape the examples (#2105)
Kong/docs.konghq.com: app/_hub/kong-inc/request-validator/index.md

publisher: Kong Inc.
version: 1.3-x

> desc: Validates requests before they reach the Upstream Service

Reviewer: ```suggestion
desc: Validates requests before they reach the Upstream service
```

> desc: Validates requests before they reach the Upstream Service

Reviewer: ```suggestion
desc: Validates requests before they reach the Upstream service
```

description: |

>   Validate requests before they reach their Upstream Service. Supports validating

Reviewer: ```suggestion
  Validate requests before they reach their Upstream service. Supports validating
```

>   Validate requests before they reach their Upstream Service. Supports validating

Reviewer: ```suggestion
  Validate requests before they reach their Upstream service. Supports validating
```

    - name: parameter_schema
      required: false
      value_in_examples:

>       description: Array of parameter validator specifications, see

Reviewer: ```suggestion
      description: Array of parameter validator specifications.
```

>         paragraph [Parameter Schema Definition](#parameter-schema-definition)

Reviewer: ```suggestion
       For details and examples, see [Parameter Schema Definition](#parameter-schema-definition).
```

>         for details and examples.

Reviewer: ```suggestion
        
```

### Further References

> The Kong schema validation format is based on the plugin schemas, for more

Reviewer: ```suggestion
The Kong schema validation format is based on the plugin schemas.
```

> details see the Kong plugin docs on storing custom entities [here][schema-docs].

Reviewer: ```suggestion
For more information,  see the Kong plugin docs on [storing custom entities][schema-docs].
```

> For more information,  see the Kong plugin docs on [storing custom entities][schema-docs].

Reviewer: ```suggestion
For more information, see the Kong plugin docs on [storing custom entities][schema-docs].
```
